control_panel_api/tests/test_services.py

```diff
@@ -78,8 +78,9 @@ def test_delete_bucket_policies(self, mock_detach_policy_from_entities,
 
     @patch('control_panel_api.aws.attach_policy_to_role')
     def test_apps3bucket_create(self, mock_attach_policy_to_role):
-        app = App.objects.create(slug='appslug')
-        s3bucket = S3Bucket.objects.create(name='test-bucketname')
+        app = mommy.make('control_panel_api.App', slug='appslug')
```

Comment:
Not sure what model mommy gives us in simple cases like these, but don't mind.

Reply:
Because manually creating you would have had to create certain fields that followed rules for url or whatever.  mommy does it all behind the scenes.  In this instance the tests were all failing because of invalid repo_urls etc.

Reply:
Makes sense. Thanks for the explanation James.